sdk/python/kfp/dsl/for_loop.py

```diff
@@ -180,7 +199,7 @@ def from_raw_items(
         return LoopArgument(
             items=raw_items,
             name_code=name_code,
-            channel_type=type(raw_items[0]).__name__,
+            channel_type=_get_first_element_type(raw_items),
         )
 
```

Comment:
Can you update the [release notes](https://github.com/kubeflow/pipelines/blob/master/sdk/RELEASE.md) to include this fix?